content/essentials/refs.mdx

```diff
@@ -0,0 +1,106 @@
+# Refs
+
+Refs, or references, allow you to have direct access to DOM element. 
+
+To create a `ref` in Solid, you can get a reference to an element through assigning it to avariable: 
+
+```jsx
+const myDiv = <div>My Element</div>;
+```
+
+For a more idiomatic way, Solid provides the `ref` attribute. 
```

Comment:
Aside: I believe the reason this is more idiomatic is because it is often nice to partially separate your DOM code from your data code